router/state_snek.go

```diff
@@ -328,7 +328,9 @@ func (s *state) _handleBootstrap(from, to *peer, rx *types.Frame) bool {
 	// current root, otherwise we won't be able to route back to them using
 	// tree routing anyway. If they don't match, silently drop the bootstrap.
 	root := s._rootAnnouncement()
-	if !root.Root.EqualTo(&bootstrap.Root) {
+	// TODO : Don't compare to root sequence here.
```

Comment:
For bootstraps I think we probably *do* want to check the sequence. For traffic frames perhaps not...